.github/workflows/build_applications.yml

```diff
@@ -0,0 +1,55 @@
+name: Build Applications
+
+on:
+  push:
+    branches: [ develop, release/** ]
+    paths:
+      - 'Applications/**'
+      - '.github/workflows/**'
+  pull_request:
+    branches: [ develop, release/** ]
+    paths:
+      - 'Applications/**'
+      - '.github/workflows/**'
+
+env:
+    ROCM_VERSION: 6.2
+    AMDGPU_INSTALLER_VERSION: 6.2.60200-1
+
+jobs:
+    build:
+        name: "Build Applications Examples"
+        runs-on: ubuntu-latest
+        container:
+            image: ubuntu:22.04
+        steps:
+          - uses: actions/checkout@v4
+
+          - name: Install dependencies
+            run: |
+                  apt-get update -qq &&
+                  apt-get install -y build-essential g++ glslang-tools \
+                    python3 python3-pip libglfw3-dev libvulkan-dev locales wget
+                  python3 -m pip install --upgrade pip
+                  python3 -m pip install cmake
```

Comment:
Things like cmake can be installed in various ways, in particular via the package manager and via pip. You should be prepared to handle errors if you try and install one way when the other way has previously been used.